frontend/app/forms/motion-to-waive-fees/motion-to-waive-fees.web.component.js

```diff
@@ -3,21 +3,233 @@ import { Scoped } from "kremling";
 import TextInput from "../inputs/text-input.component.js";
 import FormThatPrints from "../inputs/form-that-prints.component.js";
 import Section from "../inputs/section.component.js";
+import Select from "../inputs/select.component.js";
+import Radio from "../inputs/radio.component.js";
+import GroupSelect from "../inputs/group-select.component.js";
+import CheckBox from "../inputs/checkbox.component.js";
+import TextArea from "../inputs/text-area.component.js";
+import {
+  DistrictCourtList,
+  JusticeCourtList,
+  courtTypeOptions
+} from "../form-common-options/form-common-options.js";
 
-export default class MotionToWaiveFees_Web extends React.Component {
-  render() {
-    return (
-      <Scoped css={css}>
-        <FormThatPrints>
-          <Section name="1. Personal information">
-            <TextInput dataKey="person.firstName" label="First name" />
-            <TextInput dataKey="person.middleName" label="Middle name" />
-            <TextInput dataKey="person.lastName" label="Last name" />
-          </Section>
-        </FormThatPrints>
-      </Scoped>
-    );
+const waiveFeeRelationship = [
+  { value: "plantiffPetitioner", label: "Plantiff / Petitioner" },
+  {
+    value: "plantiffPetitionerAttorney",
+    label: "Plantiff / Petitioner's Attorney"
+  },
+  { value: "defendantRespondent", label: "Defendant / Respondent" },
+  {
+    value: "defendantRespondentAttorney",
+    label: "Defendant / Respondent's Attorney"
   }
+];
+
+export default function MotionToWaiveFees_Web({ data }) {
+  return (
+    <Scoped css={css}>
```

Comment:
In function components, it's preferred to use the `useCss()` hook instead of rendering the `<Scoped>` component. See https://kremling.js.org/api/use-css.html. Reason why is that it's easier to understand which dom elements have the css scope when you use hooks. See https://kremling.js.org/api/scoped-component.html#gotchas for how the `<Scoped>` component can be a bit confusing sometimes.

Reply:
Thanks I will look into that. Was switching over from the class components because it looks like most of the other forms are moving in that direction.

Reply:
👍